src/_functions.ps1

```diff
@@ -20,3 +20,21 @@ function Get-Validated-Platform {
         default     { throw "Unknown platform $Platform." }
     }
 }
+
+
+function Invoke-Cygwin-Setup {
+    param (
+        $SetupExePath,
+        $SetupExeArgs
+    )
+
+    # Because setup is a Windows GUI app, make it part of a pipeline
+    # to make PowerShell wait for it to exit.
+    Write-Host $SetupExePath $SetupExeArgs
+    & $SetupExePath $SetupExeArgs | Out-Default
+
+    # Check the exit code.
+    if ($LASTEXITCODE -ne 0) {
```

Comment:
I wonder if this can/should be LastExitCode, since StudlyCaps seems to be the style for PowerShell?

Reply:
Microsoft documents it as [`LASTEXITCODE`](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_automatic_variables?view=powershell-7.5#lastexitcode).